mobile/examples/super_resolution/ios/ORTSuperResolution/ORTSuperResolution/ContentView.swift

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+
+import SwiftUI
+
+struct ContentView: View {
+    @State private var performSuperRes = false
+    
+    func runOrtSuperResolution() -> UIImage? {
+        do {
+            let outputImage = try ORTSuperResolutionPerformer.performSuperResolution()
+            return outputImage
+        } catch let error as NSError {
+            print("Error: \(error.localizedDescription)")
+            return nil
+        }
+    }
+    
+    var body: some View {
+        ScrollView {
+            VStack {
+                VStack {
+                    Text("ORTSuperResolution").font(.title).bold()
+                        .frame(width: 400, height: 80)
+                        .border(Color.purple, width: 4)
+                        .background(Color.purple)
+                    
+                    Text("Input low resolution image: ").frame(width: 350, height: 40, alignment:.leading)
+                    
+                    Image("cat_224x224").frame(width: 250, height: 250)
+                    
+                    Button("Perform Super Resolution") {
+                        performSuperRes.toggle()
+                    }
+                    
+                    if performSuperRes {
+                        Text("Output high resolution image: ").frame(width: 350, height: 40, alignment:.leading)
+                        
+                        let outputImage = runOrtSuperResolution()
+                        if (outputImage != nil) {
```

Comment:
can use swift optional binding:
```suggestion
                        if let outputImage = runOrtSuperResolution() {
```